### DISCUSSION TITLE
Child runner are being putted in a stopped state while the parent runner is still active and running

### DISCUSSION THREAD

**Discussion by ravirkd** [npalm/terraform-aws-gitlab-runner#1051] [Q&A]
Hello,
We are seeing a weird scenario where Child runner are being putted in a stopped state while the parent runner is still active and running. The codebase doesn't have any such conditions where status of running child runner changed to stopped.
Any pointers why this case is happening?
Thanks,
Ravinder Kumar

**Comment by kayman-mk** [comment]
Nope, never seen this before. Are you using spot instances? I know that there is a "Shutdown behavior" setting for EC2 instances. Not sure where this can be checked. I didn't found it in the details of my EC2 instances.
